packages/discord.js/src/util/BitField.js

```diff
@@ -141,26 +114,19 @@ class BitField {
    * @returns {number|bigint}
    */
   static resolve(bit) {
+    console.log(bit);
     const { defaultBit } = this;
     if (typeof defaultBit === typeof bit && bit >= defaultBit) return bit;
     if (bit instanceof BitField) return bit.bitfield;
     if (Array.isArray(bit)) return bit.map(p => this.resolve(p)).reduce((prev, p) => prev | p, defaultBit);
     if (typeof bit === 'string') {
-      if (typeof this.FLAGS[bit] !== 'undefined') return this.FLAGS[bit];
+      // If (typeof this.FLAGS[bit] !== 'undefined') return this.FLAGS[bit];
```

Comment:
Ditto, or maybe we want to make a static method for resolving the strings to bits, ala channel types and the rest?